### PR TITLE
Feature/mqtt configuration retrieval with subscription

### PR DESCRIPTION
**Pull request by mgaggero**
Implements "subscribe" mechanism for MQTT binding configuration retrieval, completes #361.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -295,6 +295,12 @@ config.iota = {
     defaultType: 'Thing'
 };
 
+/**
+ * flag indicating whether the incoming notifications to the IoTAgent should be processed using the bidirectionality
+ * plugin from the latest versions of the library or the JSON-specific configuration retrieval mechanism.
```
Comment (fgalan) [inline config.js]: JSON-specific -> UL-specific ?

Comment (mgaggero) [reply config.js]: Of, course 😄

Comment (fgalan) [reply config.js]: Could you fix it, please? :)

Comment (mgaggero) [reply config.js]: Done :)

**Comment by mgaggero** [comment]
Hi @fgalan , can you try to re-run the travis-ci check? On my travis all the checks on the PR branch run successfully...

**Comment by mgaggero** [comment]
Thank you!

**Pull request merged**